views/address.html

```diff
@@ -115,11 +115,9 @@ <h1 class="modal-title" id="connectSubAddressTitle"><img src="/assets/images/inf
 									enableticketbuyer=true <br>
 									pooladdress={{ .User.UserFeeAddr }} <br>
 									poolfees={{ .PoolFees }} <br>
-									;; DEPRECATED -- use ticketbuyer.votingaddress instead <br>
-									;; ticketaddress={{ .User.MultiSigAddress }} <br>
 									[Ticket Buyer Options] <br>
 									ticketbuyer.votingaddress={{ .User.MultiSigAddress }} <br>
-									ticketbuyer.maxpriceabsolute=100
+									ticketbuyer.maxpriceabsolute=200
```

Comment:
iirc --ticketbuyer.maxpriceabsolute was deprecated after the stakediff fork when ticketbuyer was overhauled. dcrwallet will not start when that option is set due to it being invalid.  This should probably be removed.

Reply:
Good catch. I've updated the PR